src/cdk-experimental/menu/menu-group.ts

```diff
@@ -6,8 +6,19 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Directive, Output, EventEmitter} from '@angular/core';
-import {MenuItem} from './menu-item-interface';
+import {
+  Directive,
+  Output,
+  EventEmitter,
+  ContentChildren,
+  AfterContentInit,
+  QueryList,
+  OnDestroy,
```

Comment:
From a formatting perspective, I've not seen TS imports done using vertically like this -- the usual line length restrictions don't apply, at least for code I've worked on.

Again, if this is a CDK convention, you should follow it.

Reply:
ya, [some can be quite long](https://github.com/angular/components/blob/master/src/material/slider/slider.ts#L11-L57)